core/tests/protractor_desktop/navigation.js

```diff
@@ -220,6 +220,13 @@ describe('Static Pages Tour', function() {
       by.css('.protractor-test-thanks-page')).isPresent()).toBe(true);
   });
 
+  it('should visit the Volunteer page', async function() {
+    await browser.get('/volunteer');
+    await waitFor.pageToFullyLoad();
```

Comment:
This function waits for loading indicators to disappear. From the video in the PR description, it doesn't look like this page has any loading indicators, so this function does nothing.

Instead, you could replace the expectation below with a `waitFor.visibilityOf` call. That will give the page time to load but fail the test if the page never loads

Reply:
Done.